addon/model.js

```diff
@@ -741,6 +741,12 @@ export default class MegamorphicModel extends EmberObject {
     }
     return this._errors;
   }
+
+  set errors(value) {
+    if (this._schema.useUnderlyingErrorsValue(this._modelName)) {
+      return this.setUnknownProperty('errors', value);
+    }
```

Comment:
1. can we confirm the semantics of setting `errors` prior to the getter being introduced?  My expectation is that it would update the underlying errors object in place

2. assuming that's right, we should have an else branch here that has those semantics for the case where `useUnderlyingErrorsValue(modelName) === false`

Reply:
good point. I added a test and an else branch.